app/Libraries/Discord.php

```diff
@@ -118,6 +123,18 @@ private function findRole(string $roleName): int
         return (int) $role_id;
     }
 
+    public function getUserInformation(Account $account)
+    {
+        if (! $account->discord_id) {
+            return null;
+        }
+
+        return Cache::remember($account->id.'.discord.userdata', 84600, function () use ($account) {
```

Comment:
This is fine but it may read more simply if you pass in a carbon instance instead of `84600`. E.g. `now()->addDay()`.

Ref: https://laravel.com/docs/8.x/helpers#method-now
Ref 2: https://laravel.com/docs/8.x/cache#storing-items-in-the-cache

Reply:
How didn't I realise this lol https://github.com/VATSIM-UK/core/pull/2029/commits/bacf74390e917a543d3d37b3513e289f79bdac1a